src/FluentValidation/Validators/CustomValidator.cs

```diff
@@ -21,10 +21,9 @@ public class CustomValidator<T> : PropertyValidator {
 		public CustomValidator(Action<T, CustomContext> action) : base(string.Empty) {
 			_isAsync = false;
 			_action = action;
-			_asyncAction = async (x, ctx, cancel) => {
-				_action(x, ctx);
-			};
-		}
+
+            _asyncAction = (x, ctx, cancel) => Task.Run(() => action(x, ctx), cancel);

```

Comment:
Indentation should use tabs rather than spaces for consistency with rest of the project

Reply:
done